src/Routes/DetailPage/DetailPage.tsx

```diff
@@ -132,14 +133,45 @@ const DetailPage = () => {
     return <NoPermissions />;
   }
 
+  let domainTitle: React.ReactNode = domain?.title;
+  if (!domainTitle) {
+    domainTitle = <Skeleton width="15em" />;
+  }
```

Comment:
```suggestion
  const domainTitle: React.ReactNode = domain?.title === undefined ? <Skeleton width="15em" /> : domain.title;
```
I suggest the above change, as I have observed that setting an empty title evoked the skeleton was displayed for the tittle in the detail page.

Reply:
Thanks I didn't recall about HMS-2979